src/providers/vod/vod-provider.ts

```diff
@@ -76,7 +76,7 @@ export class VodProvider extends Provider {
         return {
           id: viewChangeCuePoint.id,
           startTime: viewChangeCuePoint.startTime / 1000,
-          endTime: viewChangeCuePoint.endTime,
+          endTime: viewChangeCuePoint.endTime || Number.MAX_SAFE_INTEGER,
```

Comment:
Maybe it is more correct to do in a earlier stage?
here:
https://github.com/kaltura/playkit-js-kaltura-cuepoints/blob/master/src/providers/vod/response-types/kaltura-code-cue-point.ts#L31

Reply:
no, don't mix initial data and prepared. Here we process and set absent data for cue points: for view-change - start\end time,  for Thumbs - url, etc...